satellite/accountingcache/accountingcache.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package accountingcache
```

Comment:
im curious why we would add this package here versus in `pkg/accounting` like the `tally` and `rollup` service?

Reply:
`pkg/accounting` seems like the right place by its name alone, but in code it looks to me like it's pretty tightly coupled with the accounting part of satelliteDB, and this is entirely separate from that. Secondly, I've been told that `pkg/` is for things that apply to storj code in general, not any one specific subsystem, and this code definitely only applies to a satellite. But everything is in flux anyway, so I could probably be convinced that `pkg/accounting` is where it needs to be for now.

Reply:
moved to pkg/accounting/live.